modules/client-handler/src/main/java/org/apache/ignite/client/handler/ClientInboundMessageHandler.java

```diff
@@ -0,0 +1,484 @@
+                    .packBinaryHeader(0) // Features.
+                    .packMapHeader(0); // Extensions.
+
+            write(packer, ctx);
+        } catch (Throwable t) {
+            packer = getPacker();
+
+            ProtocolVersion.LATEST_VER.pack(packer);
+            packer.packInt(ClientErrorCode.FAILED).packString(t.getMessage());
+
+            write(packer, ctx);
+        }
+    }
+
+    private void write(ClientMessagePacker packer, ChannelHandlerContext ctx) {
+        var buf = packer.toMessageBuffer().sliceAsByteBuffer();
+
+        ctx.writeAndFlush(buf);
+    }
+
+    private void writeError(int requestId, Throwable err, ChannelHandlerContext ctx) {
+        try {
+            assert err != null;
+
+            ClientMessagePacker packer = getPacker();
+            packer.packInt(ServerMessageType.RESPONSE);
+            packer.packInt(requestId);
+            packer.packInt(ClientErrorCode.FAILED);
+
+            String msg = err.getMessage();
+
+            if (msg == null)
+                msg = err.getClass().getName();
+
+            packer.packString(msg);
+
+            write(packer, ctx);
+        } catch (Throwable t) {
+            exceptionCaught(ctx, t);
+        }
+    }
+
+    private ClientMessagePacker getPacker() {
+        return new ClientMessagePacker();
+    }
+
+    private ClientMessageUnpacker getUnpacker(ByteBuffer buf) {
+        return new ClientMessageUnpacker(new ByteBufferInput(buf));
+    }
+
+    private void processOperation(ChannelHandlerContext ctx, ClientMessageUnpacker unpacker, ClientMessagePacker packer) throws IOException {
+        var opCode = unpacker.unpackInt();
+        var requestId = unpacker.unpackInt();
+
+        packer.packInt(ServerMessageType.RESPONSE)
+                .packInt(requestId)
+                .packInt(ClientErrorCode.SUCCESS);
+
+        try {
+            var fut = processOperation(unpacker, packer, opCode);
+
+            if (fut == null) {
+                // Operation completed synchronously.
+                write(packer, ctx);
+            } else {
+                fut.whenComplete((Object res, Object err) -> {
+                    if (err != null)
+                        writeError(requestId, (Throwable) err, ctx);
+                    else
+                        write(packer, ctx);
+                });
+            }
+
+        } catch (Throwable t) {
+            writeError(requestId, t, ctx);
+        }
+    }
+
+    private CompletableFuture processOperation(ClientMessageUnpacker unpacker, ClientMessagePacker packer, int opCode)
+            throws IOException {
+        // TODO: Handle all operations asynchronously (add async table API).
+        switch (opCode) {
+            case ClientOp.TABLE_DROP: {
+                var tableName = unpacker.unpackString();
+
+                ignite.tables().dropTable(tableName);
+
+                break;
+            }
+
+            case ClientOp.TABLES_GET: {
```

Comment:
Unnecessary brace

Reply:
I thought we agreed to keep them:
https://lists.apache.org/thread.html/rbdd496188264202110cec06dba8abddca844d8d24444c0842e0b29f3%40%3Cdev.ignite.apache.org%3E

Reply:
Sorry, but I'm again relying on https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines#CodingGuidelines-Whitespacesandemptylines .
And in ClientInboundMessageHandler.java you doesn't use braces for multi line cases.

Reply:
Looks like I misread the original comment and related code. This is about multi-line block in a switch statement. I don't think our coding guidelines have any guidance on this topic.

In this particular case, removing braces from all case blocks will cause variable names conflicts.